src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Circle{"
+                + "radius=" + radius
+                + "color=" + getColor()
+                + "}";
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(this);
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
```

Comment:
The parameter type for the setRadius method should be consistent with the type of the radius field, which is double. Consider changing the parameter type from int to double.